.github/workflows/build.yaml

```diff
@@ -10,6 +10,17 @@ permissions:
 name: build-test
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4
+      - name: Set up Go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5
+        with:
+          go-version: 1.20.x
+      - run: go env
+      - name: go lint
+        run: make lint-go
```

Comment:
can't we add lint also  just before the go build below in the workflow :thinking: